ssv/spectest/tests/runner/postconsensus/invalid_quorum_then_valid_quorum_sc.go

```diff
@@ -0,0 +1,288 @@
+package postconsensus
+
+import (
+	"github.com/attestantio/go-eth2-client/spec"
+
+	"github.com/bloxapp/ssv-spec/qbft"
+	"github.com/bloxapp/ssv-spec/ssv"
+	ssvcomparable "github.com/bloxapp/ssv-spec/ssv/spectest/comparable"
+	"github.com/bloxapp/ssv-spec/types"
+	"github.com/bloxapp/ssv-spec/types/testingutils"
+	"github.com/bloxapp/ssv-spec/types/testingutils/comparable"
+)
+
+// invalidThenQuorumSyncCommitteeContributionSC returns state comparison object for the invalid then quorum SyncCommitteeContribution versioned spec test
+// runner should finish since quorum is achieved
+func invalidQuorumThenValidQuorumSyncCommitteeContributionSC() *comparable.StateComparison {
```

Comment:
Just a friendly reminder that SC files were replaced by generated jsons.
However if you think this is a core test (looks like one) and that coding out the SC helps in verifying then this is great!!